_news/2019-11-27-3.2-release-notes/en.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Dodona 3.2"
+description: "Release notes Dodona 3.2"
+slug: "dodona-3.2"
+date: "2019-11-27"
+lang: en
+---
+
+The release of Dodona 3.2 brings a handy navigation menu to the exercise page, a completely renewed way to download submissions and brings back the scoresheet for a course.
+{: .fs-6 .fw-300 }
+
+## Navigation for exercises
+
+From now on, if you view an exercise through an exercise series, there will be a menu in the right margin that you can use to easily switch to the other exercises in that series.
+
+![Navigation for exercises](/assets/img/news/dodona-3.2/exercise-navigation-nl.png)\
+
+## Export of submissions
+
+The export of submissions received a full make-over. Via a wizard you can now download all your submissions per course, per series or just all of them. As a course admin you can do this for all users of your course. This new tool has a few options so that you can get started immediately with the archive you download.
```

Comment:
```suggestion
The export of submissions received a full make-over. Using a wizard, you can now download all your submissions per course, per series or just all of them. As a course admin, you can do this for all users of your course. This new tool has a few options so that you can get started immediately with the archive you download.
```